src/sql/user_insert_update.sql

```diff
@@ -0,0 +1,22 @@
+INSERT INTO slack_user (
```

Comment:
Just so ya know, it's possible to use a CTE here as well.
```
WITH team AS (
  SELECT id FROM slack_team WHERE token = ${token}
) INSERT ... (SELECT id FROM team)
```

If we did that it would actually add an additional query overall so it likely doesn't make sense. Mainly I just wanted to call this out as being possible!